app/(dashboard)/item/[id]/page.tsx

```diff
@@ -26,9 +24,9 @@ const DisplayItemDetails = async ({ params }: { params: { id: string } }) => {
     if (!item || !item.profiles) {
       throw new Error('Error fetching data');
     } else {
-      // const donorEmail = item.profiles.email;
+      const donorEmail = item.profiles.email;
       const donerId: string | undefined = item.profiles.id;
-      // const title = item.item_name;
+      const title = item.item_name;
       if (
```

Comment:
I dont know I think you can go too far with this and although its very condensed I dont know that this is easier for whatever volunteer at kindly takes over this website it might be harder even we can discuss and change on another go through